src/backend/src/lib.rs

```diff
@@ -658,18 +686,116 @@ pub fn get_user_profile() -> Result<UserProfile, GetUserProfileError> {
     })
 }
 
-/// An endpoint to be called by users on first login, to enable them to
-/// use the chain fusion signer together with Oisy.
+/// Creates a new proof-of-work challenge for the caller.
+///
+/// # Errors
+/// Errors are enumerated by: `CreateChallengeError`.
+///
+/// # Returns
+///
+/// * `Ok(CreateChallengeResponse)` - On successful challenge creation.
+/// * `Err(CreateChallengeError)` - If challenge creation fails due to invalid parameters or
+///   internal errors.
+#[update(guard = "may_write_user_data")]
+#[candid_method(update)]
+pub async fn create_pow_challenge() -> Result<CreateChallengeResponse, CreateChallengeError> {
```

Comment:
I would recommend making a PR with these new canister methods with just `todo()` in the function body.  That PR would then also contain the .did file changes and the bindings.